documentation/release_notes.rst

```diff
@@ -11,14 +11,70 @@ creating efficient heterogeneous applications.
 New in 2022.10.0
 ================
 
+Deprecation Notices
+-------------------
+The ``ONEDPL_USE_AOT_COMPILATION`` and ``ONEDPL_AOT_ARCH`` CMake options are deprecated and will be removed in a future
+release. Please use the relevant compiler flags to enable this feature.
+
+New Features
+------------
+- Added parallel range algorithms in ``namespace oneapi::dpl::ranges``: ``set_intersection``, ``set_union``,
+  ``set_difference``, ``set_symmetric_difference``, ``includes``, ``unique``, ``unique_copy``, ``destroy``,
+  ``uninitialized_fill``, ``uninitialized_move``, ``uninitialized_copy``, ``uninitialized_value_construct``,
+  ``uninitialized_default_construct``, ``reverse``, ``reverse_copy``, ``swap_ranges``. These algorithms operate with
+  C++20 random access ranges.
+- Improved performance of ``gpu::inclusive_scan`` kernel template and added support for binary operator and type
+  combinations which do not have a SYCL known identity.
+- Improved performance of ``inclusive_scan_by_segment``, ``exclusive_scan_by_segment`` and set operation algorithms when
+  using device policies: ``set_union``, ``set_difference``, ``set_intersection``, ``set_symmetric_difference``.
```

Comment:
The formulation of this sentence is a bit strange.
```suggestion
- Improved performance of scan_by_segment and set operation algorithms when
  using device policies: ``inclusive_scan_by_segment``, ``exclusive_scan_by_segment``, ``set_union``, ``set_difference``, ``set_intersection``, ``set_symmetric_difference``.
```

Or alternatively, just list the algorithms out without other commentary:
``` 
- Improved performance of ``inclusive_scan_by_segment``, ``exclusive_scan_by_segment``, ``set_union``, ``set_difference``, ``set_intersection``, and ``set_symmetric_difference`` algorithms when
  using device policies.
```

Reply:
Separating the set operation algorithms out with the colon is odd even with the suggestion.  I've just listed the algorithms as you suggest.